test/e2e/petset.go

```diff
@@ -59,15 +59,18 @@ const (
 // GCE Quota requirements: 3 pds, one per pet manifest declared above.
 // GCE Api requirements: nodes and master need storage r/w permissions.
 var _ = framework.KubeDescribe("PetSet [Slow] [Feature:PetSet]", func() {
-	f := framework.NewDefaultFramework("petset")
+	options := framework.FrameworkOptions{
+		GroupVersion: &unversioned.GroupVersion{Group: apps.GroupName, Version: "v1alpha1"},
+	}
+	f := framework.NewFramework("petset", options, nil)
 	var ns string
 	var c *client.Client
 
 	BeforeEach(func() {
 		// PetSet is in alpha, so it's disabled on all platforms except GCE.
```

Comment:
fyi these tests also depend on init containers and dynamic provisioning, which we piped through annotations so they should be on. In theory someone could disable annotation alpha features thorugh something like an admission controller.